src/Modules/UngradedModules.jl

```diff
@@ -6386,13 +6384,11 @@ function free_resolution(M::SubquoModule{<:MPolyRingElem};
   singular_kernel_entry = Singular.Module(base_ring(singular_free_module),
                               [singular_free_module(repres(g)) for g in gens(kernel_entry)]...)
 
-  singular_kernel_entry.isGB = true
+  gbpres = Singular.std(singular_kernel_entry)
```

Comment:
Only `fres`  and `sres` need a precomputed Gröbner basis of the input. Maybe we should not globally do this since other `algorithm`s like `mres` will be added.

Reply:
OK will shift the line for future cases.